chain-api/src/utils/dex/format.helper.ts

```diff
@@ -16,6 +16,15 @@ import BigNumber from "bignumber.js";
 
 import { ConflictError } from "../error";
 
+/**
+ *
+ * @param address address of pool in string
+ * @returns
+ */
+export const virtualAddress = (address: string) => {
+  return "service|" + address;
+};
```

Comment:
My suggestion is generating those values with a prefix suggesting it's an identifier of a pool: `service|pool_<the rest>`

also I'm not sure about the naming here, we should probably have something like:

```suggestion
export function poolIdentityKey(poolId: string) => {
  return `service|pool_${poolId}`;
};
```

typicaly for "account" identifier on chain we use either `identityKey` or `userAlias` (`alias`). So it can be maybe also `poolAlias`? 🤔

Reply:
We have updated the function to work as `poolAlias`.